### ISSUE TITLE
Can't scroll in "Parent Page" select when too much pages are created 

### ISSUE THREAD

**Issue by saxinte** [getgrav/grav-plugin-admin#941]
Hello Grav Team :)

I just found that if you have too many pages created in your Grav App (I have more than 200 pages), then you can't found them within the "Add page" feature in the Admin panel.

Tested with Grav 1.0.10 and Grav Admin 1.0.10: bug is present
Tested with Grav 1.1.13 and Grav Admin 1.2.8: bug is present

How to replicate:
- Create a subfolder within your app in `/user/pages/a` http://imgur.com/tMe3Xyb
- Add 200 pages in it.
- Open the admin, then click on "Pages" on the Left Hand Panel.
- Click "Add Page" at the top-right.
- In the "Parent page" select, try to access a page that is not inside this `a` folder. It's impossible.

I used `a` folder in order to have all those pages displayed at the top of the "Parent page" select. Pages above can't be reached.

I can find my page in typing it directly but it's hard to find it. It's also very laggy because of the amount of DOM node here. A better solution would be either a real `<select>` element or to be able to look for a given path, not the page title.

I've added a screenshot and a video to show you the issue (In the video I show you that I only see the pages within the `/a`folder - I can't access other pages)

- Pages: http://imgur.com/tMe3Xyb
- Video: http://imgur.com/QxLKSxX

Thanks,
Sebastien

**Comment by phideas** [comment]
**Feature request:**
I can confirm this issue is a real problem. As the number of website pages grows, it is getting hard to select parent when trying to add new page. I think a new aproach to select parent is needed here. I do not know the right answer to this problem, but something that would show only the requiered top levels is needed.

**Comment by rhukster** [comment]
What browser is this?  I'm trying the parent dropdown with the latest release and it's fine even with 1000+ pages.

**Comment by phideas** [comment]
Sorry for confusion. Can't recreate this bug either. My complain is related to UX when a lot of pages in list makes it hard for user to quickly select parent. Gonna create a feature request.

**Comment by saxinte** [comment]
Hello guys, sorry being away for long, I was busy working on other stuff.
I just updated to Grav 1.2.2 and Admin 1.3.3 and even if the way Grav displays the pages and child, the issue is still their.

I've recorded a video to show you the issue. Basically, I can see my page from Alphabetic Order with the parent page and the modules within. But when I reach the end of the scroll I don't see the remaining pages.

The search page works if I'm start typing a page which is fine, but not the scroll.
Here's the video with the issue: https://streamable.com/4zhx5

Thanks
